packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/common.rs

```diff
@@ -76,11 +82,11 @@ pub struct ComponentCommonData {
     ///
     /// Macros remain in this vector only if they couldn't be expanded.
     /// TODO: implement function macros so they don't stay in this vector
-    pub children: Vec<ComponentPointerTextOrMacro>,
+    pub children: Vec<UntaggedContent>,
```

Comment:
It doesn't need to change for this PR, but I think `UntaggedContent` is a confusing type, as it isn't clear how it used to be tagged and how this is different. I'm not sure of a better word.